### PR TITLE
Feature/home page form

### LINKED ISSUE #12: Creating form for main page

### PR DESCRIPTION
**Pull request by FocalChord**
Closes #12

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,62 +1,105 @@
...
+  },
+  {
+    label: 'Light Theme',
+    value: 'light',
+  },
+];
+
+const Home: React.FC = () => {
+  const handleSubmit = (values: FormValues): void => {
+    // Make API Call with JSON.stringify(values);
```
Comment (Drewbi) [inline pages/index.tsx]: We will just be rerouting to /home/:seed so you will need to generate a random number to be used as the seed and add the options in some way, either `/home/94829483?type=e_commerce` or  `/home/e_commerce-94829483` or you could encode the type to make it less obvious what we're doing eg. `/home/MVPWG33NNVSXEY3F-94829483`

File: pages/index.tsx [hunk 2]
```diff
@@ -1,62 +1,105 @@
...
       <footer>
-        <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className="logo" />
-        </a>
+        <p> Made by Team 2 in Pod 0.3.2 </p>
```
Comment (Drewbi) [inline pages/index.tsx]:
```suggestion
        <p> Made by Team 1 in Pod 0.3.2 </p>
```
I think we are Team 1

File: pages/index.tsx [hunk 3]
```diff
@@ -67,6 +110,15 @@ export default function Home() {
           flex-direction: column;
           justify-content: center;
           align-items: center;
+          background-color: #e2dadb;
```
Comment (Drewbi) [inline pages/index.tsx]: Maybe stick with white for the background

**Review by Drewbi — CHANGES_REQUESTED** [review]
Good stuff, just a couple of ez changes for now :)

**Review by laveesh — APPROVED** [review]
:+1:

**Review by Drewbi — APPROVED** [review]

**Review by Drewbi — APPROVED** [review]

**Pull request merged**